near/omni-bridge/src/storage.rs

```diff
@@ -185,31 +198,40 @@ impl Contract {
             .saturating_mul((Self::get_basic_storage() + key_len + value_len).into())
     }
 
-    pub fn required_balance_for_init_transfer(&self) -> NearToken {
+    pub fn required_balance_for_init_transfer(&self, msg: Option<String>) -> NearToken {
```

Comment:
Could you please remind me why we need to pass the `msg` here? since the regular init transfers doesn't accept additional msg.

Reply:
Not related to this PR I supposed but we did it for more precise calculations in finalizations to 3rd chains